Src/Fido2.Models/AssertionOptions.cs

```diff
@@ -54,6 +54,12 @@ public class AssertionOptions
     [JsonPropertyName("hints")]
     public IReadOnlyList<PublicKeyCredentialHint> Hints { get; set; } = Array.Empty<PublicKeyCredentialHint>();
 
+    /// <summary>
+    /// This member is intended for use by Relying Parties that wish to select a preference regarding the attestation statement format used, if such an attestation is requested.
+    /// </summary>
+    [JsonPropertyName("attestationFormats")]
+    public IReadOnlyList<AttestationStatementFormatIdentifier> AttestationFormats { get; set; } = [];
```

Comment:
Can we extend this description to include:

Values are ordered from most preferable to least preferable. This parameter is advisory and the authenticator MAY use an attestation statement not enumerated in this parameter.